lib/reporters/mini.js

```diff
@@ -163,6 +164,21 @@ class MiniReporter {
 				}
 
 				status += '  ' + colors.title(test.title) + '\n';
+
+				if (test.logs) {
+					test.logs.forEach(log => {
+						const logLines = indentString(colors.log(log), 6);
+						const logLinesWithLeadingFigure = logLines.replace(
+							/^ {6}/,
+							`    ${colors.information(figures.info)} `
+						);
```

Comment:
This might be better as:

```js
const logLines = indentString(colors.log(log), 1, {
  indent: `    ${colors.information(figures.info)} `,
  includeEmptyLines: true
});
```

(Though that `indent` value should probably be a constant, but feel free to redeclare it in the `verbose` logger.)

See [`indent-string`'s API](https://www.npmjs.com/package/indent-string#api). Including empty lines would be good given the figure prefix.

You'll have to update the `indent-string` dependency to `3.2.0` since the `includeEmptyLines` feature is pretty new.